cameraview/src/main/java/com/otaliastudios/cameraview/CameraView.java

```diff
@@ -559,9 +559,10 @@ private void checkPermissionsManifestOrThrow(SessionType sessionType, Audio audi
                         return;
                     }
                 }
-                LOG.e("Permission error:", "When the session type is set to video,",
-                        "the RECORD_AUDIO permission should be added to the app manifest file.");
-                throw new IllegalStateException(CameraLogger.lastMessage);
+                CameraException cameraException = new CameraUnavailableException("Permission " +
```

Comment:
I would leave this as it was. This means the developer has not learnt how to use the library. It's better if he does before releasing

Reply:
I see your point. However, I got some crash reports potentially caused by users trying to change already granted permissions during runtime. I know this is very unlikely, but this way they won't annoy me anymore.